styles/base.css

```diff
@@ -278,6 +279,18 @@ header {
     }
 }
 
+/* Credit */
+.credit {
+    color: var(--text-grey);
+    font-family: var(--bs-font-monospace);
+    font-size: 12px;
```

Comment:
We use `rem` everywhere else `font-size` is declared, should probably do so here as well?

Reply:
Thanks for catching 🙏 I grabbed this CSS from Figma and thought I had it all cleaned up - missed this font unit